substrate/client/src/client.rs

```diff
@@ -154,6 +154,14 @@ pub struct JustifiedHeader<Block: BlockT> {
 }
 
 impl<Block: BlockT> JustifiedHeader<Block> {
+	/// Creates new JustifiedHeader with given header, justification and authorities.
+	pub fn new(header: <Block as BlockT>::Header, justification: ::bft::Justification<Block::Hash>, authorities: Vec<AuthorityId>,) -> Self {
+		JustifiedHeader{
```

Comment:
house style puts a space between the name of the struct and the `{`, particularly when it's not inline.

Reply:
improved